lua/weapons/arccw_base/shared.lua

```diff
@@ -298,6 +298,9 @@ SWEP.BashAng = Angle(-16.524, 70, -11.046)
 SWEP.ActivePos = Vector(0, 0, 0)
 SWEP.ActiveAng = Angle(0, 0, 0)
 
+SWEP.ReloadPos = nil
+SWEP.ReloadAng = nil
+
```

Comment:
these are analogous to activepos and such and allow you to move the weapon around during drawing/reload